### DISCUSSION TITLE
Updating to recent version on Rasbian (buster)

### DISCUSSION THREAD

**Discussion by trex005** [Taxel/PlexTraktSync#1905] [Support]
I worked on this for hours yesterday and hours again today so I am finally breaking down and hoping there is someone who can help me out.

I have a Rasbian (buster) setup.
I installed pipx and then used pipx to install PlexTraktSync as explained in the readme.  The problem is, it installed PlexTraktSync 0.26.11 and as you know from [the pinned issue](https://github.com/Taxel/PlexTraktSync/issues/1787), that version is no longer functional.

**What I've tried**
I don't know pipx well, so I don't know how it selects the version to install.
I originally had python 3.7.3 installed so I thought it might be a dependency issue so I installed python 3.12.3 to no avail.
I have fuddled around with different pipenv and pyenv methods.
I have tried to update my pipx version (I'm running 0.12.1.0) but I can't find a Raspian buster repository with anything newer.

Does anyone know of any way for me to get a functional version on Rasbian?

**Comment by glensc** [comment]
0.26.11 is last version supporting Python 3.7:
- #773

possibly your pipx is still using 3.7

you should probably then just use docker image instead on such old system.

you should be able to install newer `pipx` (than your distro) from pypi using `pip`.

**Comment by glensc** [comment]
you can try installing this branch (Install code from Pull request is described in readme):
- https://github.com/Taxel/PlexTraktSync/pull/1921

**Comment by glensc** [comment]
also, what is the output if you tell pipx version of the app you want to install?

**Comment by glensc** [comment]
user dissaappeared. closing
